lumberjack.go

```diff
@@ -449,6 +461,11 @@ func (l *Logger) max() int64 {
 	return int64(l.MaxSize) * int64(megabyte)
 }
 
+//isAfter checks if the log file creating time exceed the interval
+func (l *Logger) intervalExceeded() bool {
```

Comment:
The method name is a bit no-descriptive, how about: `exceedsRollingInterval`?

Reply:
yes, exceedsRollingInterval is better :)